build.gradle

```diff
@@ -19,89 +19,90 @@
 /*************************/
 
 buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.github.jengelman.gradle.plugins:shadow:1.2.3'
-    }
+  repositories {
```

Comment:
I strongly recommend to avoid reformatting.  Unable to review the changes exactly.

Reply:
That's a mistake reverting..